set.mm

```diff
@@ -633467,6 +633467,21 @@ deduction version (Contributed by metakunt, 22-May-2024.) $)
       UDTZUBZUBUQURUEAVAVDAUSUTJKUCAVBVCLMUCUCBEFGHIUFUGUHACULDUMSPQUIUJ $.
   $}
 
+  ${
+    casetrich.1 $e |- ( ph -> A e. RR ) $.
+    casetrich.2 $e |- ( ph -> B e. RR ) $.
+    casetrich.3 $e |- ( ( ph /\ A < B ) -> ch ) $.
+    casetrich.4 $e |- ( ( ph /\ A = B ) -> ch ) $.
+    casetrich.5 $e |- ( ( ph /\ B < A ) -> ch ) $.
+    $( Case elimination based on trichotomy of real numbers.  (Contributed by
+       metakunt, 24-May-2024.) $)
+    casetrich $p |- ( ph -> ch ) $=
```

Comment:
1. Looks a bit like ltlecasei.  I wonder whether a shorter proof based on it exists.
2. Suggest to rename ch to ps, to be more along the lines of our usual patterns.

Reply:
It is, I deleted casetrich, as I didn't want to search the database I thought it is just faster to implement it myself.